package.json

```diff
@@ -2,24 +2,23 @@
   "name": "travvy-server",
   "version": "1.0.0",
   "description": "",
-  "main": "app.js",
+  "author": "",
   "scripts": {
-    "start": "node travvy-server/app.js",
     "test": "mocha",
+    "lint": "./node_modules/.bin/eslint --fix **/*.js",
     "dev": "node ./node_modules/nodemon/bin/nodemon.js travvy-server/app.js --exec \"npm run lint && node\"",
-    "lint": "./node_modules/.bin/eslint --fix **/*.js"
+    "start": "node travvy-server/app.js"
   },
-  "keywords": [],
-  "author": "",
-  "license": "ISC",
   "dependencies": {
     "body-parser": "^1.19.0",
     "cors": "^2.8.5",
     "express": "^4.17.1",
+    "firebase": "^8.1.1",
```

Comment:
Nice to see the use of a non relational database for storage of images, glad to see it working